src/test/kotlin/com/zepben/evolve/services/network/tracing/FindWithUsagePointsTest.kt

```diff
@@ -26,11 +26,11 @@ import org.junit.jupiter.api.extension.RegisterExtension
 import java.util.function.Consumer
 
 
-class FindWithUsagePointsTest {
+internal class FindWithUsagePointsTest {
 
     @JvmField
     @RegisterExtension
-    var systemOutRule: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+    var systemOut: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
```

Comment:
todo: wont comment on any more, but all `SystemLogExtension.SYSTEM_ERR` should become `systemErr` not `systemOut`

Reply:
Renamed as such. This should be the last one.